### DISCUSSION TITLE
Link Not Working: New learning console course: Automate and scale with the Amazon Ads API

### DISCUSSION THREAD

**Discussion by Deepak53875387** [amzn/ads-advanced-tools-docs#301] [Q&A]
The link corresponding to title: New learning console course: Automate and scale with the Amazon Ads API not wokring :
LINK:https://learningconsole.amazonadvertising.com/student/collection/7020/path/61094/activity/61062

Can someone give me the new link

**Comment by jerirose** [comment]
Hi @Deepak53875387 , thank you for pointing this out! We've removed that tile for next week's content update.

The learning console team updates their content regularly, and they sometimes retire older content, so there may not be a 1-for-1 replacement for this console course, unfortunately. [This video](https://learningconsole.amazonadvertising.com/student/collection/44322/path/93818/activity/14995) might offer a similarly high-level overview.
